rules.d/60-aspm.rules

```diff
@@ -0,0 +1,20 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="aspm_end"
+
+# Tiger Lake VMD
+SUBSYSTEMS=="pci", DRIVERS=="vmd", ATTRS{vendor}=="0x8086", ATTRS{device}=="0x9a0b", SUBSYSTEM=="pci", DRIVER=="nvme", GOTO="aspm_enable"
+
+GOTO="aspm_end"
+
+LABEL="aspm_enable"
+
+TEST=="link/clkpm", ATTR{link/clkpm}="1"
+TEST=="link/l0s_aspm", ATTR{link/l0s_aspm}="1"
+TEST=="link/l1_aspm", ATTR{link/l1_aspm}="1"
+TEST=="link/l1_1_aspm", ATTR{link/l1_1_aspm}="1"
+TEST=="link/l1_1_pcipm", ATTR{link/l1_1_pcipm}="1"
+TEST=="link/l1_2_aspm", ATTR{link/l1_2_aspm}="1"
+TEST=="link/l1_2_pcipm", ATTR{link/l1_2_pcipm}="1"
```

Comment:
Is this all the links there are? Is this list the same for all (potential) devices? Some kind of glob assignment would probably be better.